core/controllers/classifier_test.py

```diff
@@ -67,9 +65,18 @@ def setUp(self):
             '_c_lw': [],
             '_c_l': []
         }
-        self.job_id = classifier_services.create_classifier_training_job(
-            algorithm_id, interaction_id, self.exp_id, self.exploration.version,
-            'Home', training_data, feconf.TRAINING_JOB_STATUS_PENDING)
+        classifier_training_job = (
+            classifier_services.get_classifier_training_job(
+                self.exp_id, self.exploration.version, 'Home'))
+        self.job_id = classifier_training_job.job_id
+        # Use the mark_training_job_pending function below after this is merged
```

Comment:
Just to check, what is the merge procedure here? Should Giritheja merge his PR first before this PR can be merged?